Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2005-09-02
Initial Package Version: 2.6.12
Origin: Jim Gifford
Upstream Status: Unknown
Description: Fixes Compile Issue with GCC 4.x
 
--- linux-2.6.13/drivers/sbus/char/bbc_envctrl.c.orig	2005-09-02 18:39:02.000000000 +0000
+++ linux-2.6.13/drivers/sbus/char/bbc_envctrl.c	2005-09-02 18:39:07.000000000 +0000
@@ -13,7 +13,6 @@
 #include <linux/delay.h>
 #include <asm/oplib.h>
 #include <asm/ebus.h>
-static int errno;
 #include <asm/unistd.h>
 
 #include "bbc_i2c.h"
